selfdrive/ui/spinner

```diff
@@ -1,12 +1,19 @@
 #!/bin/sh
 
-if [ -f /EON ]; then
+if [ -f /EON ] && [ -f ../QCOM_QT ]; then
   export LD_LIBRARY_PATH="/system/lib64:$LD_LIBRARY_PATH"
-  exec ./android/spinner/spinner "$1"
+  FILE="./android/spinner/spinner"
 else
-  if [ -f /TICI ] && [ ! -f qt/spinner ]; then
-    cp qt/spinner_aarch64 qt/spinner
+  if [ -f /EON ]; then
```

Comment:
this doesn't work as intended anymore, let's leave spinner and text alone for now